RIOT-OS/RIOT
# tests/pkg_tinydtls_sock_async: fix stack size for event thread

### Contribution description
The stack size of `event_thread_medium` is way to small to handle DTLS, so let's increase it ;-).



### Testing procedure
Use two `samr21-xpro` or `iotlab-m3` and flash `tests/tests/pkg_tinydtls_sock_async` to both of them. Start a terminal to each. Run `dtlss start` on one and send a DTLS message from the other `dtlsc <server address> "Hello World!"`.

With this patch it should succeed. Currently it crashes.



### Issues/PRs references
Found in https://github.com/RIOT-OS/RIOT/pull/14906#issuecomment-684797218
